packages/moonstone/BodyText/tests/BodyText-specs.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {mount, shallow} from 'enzyme';
+import BodyText from '../BodyText';
+import {MarqueeDecorator} from '../../Marquee';
```

Comment:
`shallow` and `MarqueeDecorator` aren't used, could probably clean these up.

Reply:
done